lib/Dmv_Facility_Factory.rb

```diff
@@ -0,0 +1,35 @@
+class DmvFacilityFactory 
```

Comment:
Remember that our naming convention for files names is snake case, so this file should be `dmv_facility_factory.rb`